index.js

```diff
@@ -0,0 +1,58 @@
+import Library from './modules/book.js';
+import showList from './modules/showList.js';
+import showAddNew from './modules/showAddNew.js';
+import showContact from './modules/showContact.js';
+import { DateTime } from './modules/luxon.js';
+
+// The funtions below display the header
+showList();
+showAddNew();
+showContact();
+
+// Display the time
+setInterval(() => {
+  const currentDate = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
+  document.getElementById('date').innerHTML = currentDate;
+}, 1000);
+
+const booklist = document.querySelector('#booklist');
+const inputTitle = document.getElementById('title');
+const inputAuthor = document.getElementById('author');
+const buttonAdd = document.getElementById('add_button');
+
+const obj = new Library();
+
+function printBooks() {
+  const shelf = JSON.parse(localStorage.getItem('shelf')) || [];
+  let innerhtml = '';
```

Comment:
- [ ] Kindly use the ES6 arrow syntax for functions in your code to meet the project requirements.